### PR TITLE
Fix item image after edition

### LINKED ISSUE #106: Fix item image not displaying after edition

### PR DESCRIPTION
**Pull request by bafavre**
Fixes #106

Before the item image URI would be lost if we would edit the item without changing the image.

### CODE REVIEW

File: app/src/main/java/com/example/sharingang/EditItemFragment.kt [hunk 1]
```diff
@@ -77,7 +77,7 @@ class EditItemFragment : Fragment() {
                     categoryString = resources.getStringArray(R.array.categories)[binding.categorySpinner.selectedItemPosition],
                     latitude = binding.latitude?.toDoubleOrNull() ?: 0.0,
                     longitude = binding.longitude?.toDoubleOrNull() ?: 0.0,
-                    imageUri = imageUri?.toString()
+                    imageUri = if(imageUri != null) imageUri.toString() else existingItem.imageUri
```
Comment (cronokirby) [inline app/src/main/java/com/example/sharingang/EditItemFragment.kt:80]:
```suggestion
                    imageUri = imageUri?.toString() ?: existingItem.imageUri
```

Pretty sure that this works, and is more idiomatic kotlin

**Review by cronokirby — CHANGES_REQUESTED** [review]

**Comment by cronokirby** [comment]
Nice catch otherwise!

**Review by betrisey — APPROVED** [review]

File: app/src/main/java/com/example/sharingang/EditItemFragment.kt (hunk 1 above)
Comment (bafavre) [reply app/src/main/java/com/example/sharingang/EditItemFragment.kt]: Yeah you're right, I tried to do something like that but failed at first, thanks!

**Review by cronokirby — APPROVED** [review]
Cool beans!

**Pull request merged**